junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/DiscoveryFilterApplier.java

```diff
@@ -90,10 +90,10 @@ private boolean includeClass(ClassTestDescriptor classTestDescriptor, List<Class
 		Class<?> testClass = classTestDescriptor.getTestClass();
 
 		// @formatter:off
-		return classNameFilters.stream()
-				.map(filter -> filter.apply(testClass.getName()))
-				.noneMatch(FilterResult::excluded);
-		// @formatter:on
+        return classNameFilters.stream()
+                .map(filter -> filter.apply(testClass.getName()))
+                .noneMatch(FilterResult::excluded);
+        // @formatter:on
```

Comment:
Huh, I _did_ rebase it, so obviously I did something wrong. I'll fix this in a future commit.